Byron's submission/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset='utf-8' />
+    <title>Byron's Story Map (Oct 2024 Sales Exercise)</title>
+    <meta name='viewport' content='initial-scale=1,maximum-scale=1,user-scalable=no' />
```

Comment:
**suggestion:** There are a few accessibility issues that came up when I scanned the map.

![Axe dev tools issues](https://github.com/user-attachments/assets/6cc1bcf6-c6df-4e95-8763-b32f4396ab36)

Many of them have to do with color contrast, which is more than likely related to the non-active steps being transparent, so I wouldn't worry about those. However, I would address the last two in my HTML -- a [`lang` attribute on the `html` element](https://dequeuniversity.com/rules/axe/4.10/html-has-lang?application=AxeChrome), and not using [`user-scalable=no` in the viewport meta tag](https://dequeuniversity.com/rules/axe/4.10/meta-viewport?application=AxeChrome).
```suggestion
<html lang="en">

<head>
    <meta charset='utf-8' />
    <title>Byron's Story Map (Oct 2024 Sales Exercise)</title>
    <meta name='viewport' content='initial-scale=1,maximum-scale=1' />
```